openmdao/core/system.py

```diff
@@ -5333,3 +5321,17 @@ def _get_jac_col_scatter(self):
             tarr -= toffset
 
         return sarr, tarr, tsize, has_dist_data
+
+    def _resolve_src_shape(self, input_name):
```

Comment:
Am I missing something, or does this do nothing?

Reply:
Oops, leftover code from some aborted early iteration.  I removed it.